tests/stdlib/tos.nim

```diff
@@ -26,6 +26,31 @@ Raises
 # test os path creation, iteration, and deletion
 
 import os, strutils, pathnorm
+import "$nim/compiler/unittest_light"
+
+template runTestCases*(msg: string, examples, body: untyped): bool =
+  ##[
+  Runs body on each example (input, expected).
+  Takes care of calling unixToNativePath.
+  Returns true on success.
+  ]##
+  runnableExamples:
```

Comment:
Who runs ``nim doc tests/stdlib/tos.nim`` in order to check this runnable example? Not me.

Reply:
fixed, moved to a test in a block